script/v2/fork-scenarios/scUSDCv2Rebalance.s.sol

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.13;
+
+import "forge-std/console2.sol";
+import "forge-std/Test.sol";
+
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {WETH} from "solmate/tokens/WETH.sol";
+import {AccessControl} from "openzeppelin-contracts/access/AccessControl.sol";
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+
+import {Constants as C} from "../../../src/lib/Constants.sol";
+import {Swapper} from "../../../src/steth/Swapper.sol";
+import {PriceConverter} from "../../../src/steth/PriceConverter.sol";
+import {scWETHv2} from "../../../src/steth/scWETHv2.sol";
+import {scUSDCv2} from "../../../src/steth/scUSDCv2.sol";
+import {AaveV2ScUsdcAdapter} from "../../../src/steth/scUsdcV2-adapters/AaveV2ScUsdcAdapter.sol";
+import {AaveV3ScUsdcAdapter} from "../../../src/steth/scUsdcV2-adapters/AaveV3ScUsdcAdapter.sol";
+import {MainnetDeployBase} from "../../base/MainnetDeployBase.sol";
+import {IAdapter} from "../../../src/steth/IAdapter.sol";
+
+/**
+ * A script exercising the rebalance functionality of scUSDCv2 in different situations on a forked mainnet.
+ * cmd: forge script script/v2/fork-scenarios/scUSDCv2Rebalance.s.sol --skip-simulation -vv
+ */
+contract scUSDCv2Rebalance is MainnetDeployBase, Test {
+    using FixedPointMathLib for uint256;
+
+    PriceConverter priceConverter;
+    scWETHv2 scWethV2;
+    scUSDCv2 scUsdcV2;
+
+    IAdapter aaveV3Adapter;
+    IAdapter aaveV2Adapter;
+
+    address alice = address(0x0100);
+    address bob = address(0x0200);
```

Comment:
we have an alice and bob in the constants library in my PR

I can consolidate when I pull?